src/init.c

```diff
@@ -353,6 +353,6 @@ SEXP initLastUpdated(SEXP var) {
 
 SEXP dllVersion(void) {
   // .onLoad calls this and checks the same as packageVersion() to ensure no R/C version mismatch, #3056
-  return(ScalarString(mkChar("1.14.99")));
+  return(ScalarString(mkChar("1.15.0")));
```

Comment:
I thought we did not want 1.15.0 (or any even number release version) to get on master branch?

Reply:
For the tag, it must be changed to match the tag itself. This matches the release to CRAN so should not produce any issues. However, not sure if this should be merged to master (rather the 1.15.99).

Reply:
yes I think it is normal to have the version number match the tag name, but I thought we kept that code on a separate branch? (not master) 
correct me if I'm wrong.

Reply:
I think that is what we did previously, so yes, let's not include this last commit in the PR. I'll remove those changes from this PR.